Bulldozer.BinaryFile/Maps/PersonDocument.cs

```diff
@@ -275,27 +280,15 @@ public int ProcessFiles( List<ZipArchiveEntry> importFiles, RockContext rockCont
                             .ToDictionary( a => a.Key, v => v.Value.Value ).ToJson();
                     }
 
-                    ProviderComponent storageProvider;
-                    if ( rockFile.StorageEntityTypeId == DatabaseProvider.EntityType.Id )
+                    if ( rockFile.StorageProvider == null )
```

Comment:
Oh, did you test this? I thought it might still be necessary to "Load" the storage provider, as seen here: https://github.com/SparkDevNetwork/Rock/blob/a37c5999316c814cef4de2b1bca7de8e60915b2f/Rock/Bulk/SampleData/SampleDataManager.cs#L3313C17-L3313C89 that would be interesting if you don't have to load it from the id, but you might not.

Reply:
I looked at just the commit, https://github.com/KingdomFirst/Bulldozer/pull/111/commits/0d09696bf5ec68c75ce334f587c687ac95b67285 and I didn't realize you were already doing what it was doing in their example. Never mind me...